Sources/SPMBuildCore/BuildParameters.swift

```diff
@@ -391,18 +391,22 @@ public struct BuildParameters: Encodable {
         return try buildPath.appending(binaryRelativePath(for: product))
     }
 
+    /// Returns the path to the dynamic library of a product for the current build parameters.
+    internal func potentialDynamicLibraryPath(for product: ResolvedProduct) throws -> RelativePath {
```

Comment:
nit: I don't think we use explicit `internal` in our codebase

```suggestion
    func potentialDynamicLibraryPath(for product: ResolvedProduct) throws -> RelativePath {
```

Reply:
done!